rush-plugins/rush-serve-plugin/README.md

```diff
@@ -0,0 +1,14 @@
+# @rushstack/rush-serve-plugin
+
+A Rush plugin that hooks into action execution and runs an express server to serve project outputs. Meant for use with watch-mode commands.
```

Comment:
Add a note such as `UNDER DEVELOPMENT`  to clarify that this feature is not ready for general usage yet.

Reply:
Added.